plugins/services/src/js/columns/ServicesTableStatusColumn.tsx

```diff
@@ -1,15 +1,16 @@
 import * as React from "react";
 import { Trans } from "@lingui/macro";
 import { TextCell } from "@dcos/ui-kit";
+import classnames from "classnames";
 
 import StringUtil from "#SRC/js/utils/StringUtil";
-import ServiceStatusIcon from "../components/ServiceStatusIcon";
+import { SortDirection } from "#PLUGINS/services/src/js/types/SortDirection";
+import ServiceStatusProgressBar from "#PLUGINS/services/src/js/components/ServiceStatusProgressBar";
```

Comment:
I don't know why but webpack would intermittently fail to find this module and not compile when this was a relative path vs using an absolute path with `#PLUGINS`, so that is why this is changed.

Reply:
For me, ` import ServiceStatusProgressBar from "../components/ServiceStatusProgressBar";` works.